src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -1381,6 +1381,14 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "Home_Alert_RiskStatusLowered_PrimaryButtonTitle" = "OK";
 
+/* Home Alert another high exposure */
+
+"Home_Alert_RiskStatusAnotherHighExposure_Title" = "Neue Risikobegegnung";
+
+"Home_Alert_RiskStatusAnotherHighExposure_Message" = "Sie hatten eine erneute Begegnung mit erhöhtem Risiko. Alle Risikobegegnungen der vergangenen 10 Tage finden Sie in Ihrem Kontakttagebuch.";
```

Comment:
"...alle Risikobegegnungen der vergangenen 10 Tage..." > shouldn't the number of days be a variable maybe that is configurable?  Or is it always 10 days regardless of what the relevant exposure time is (see https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12065)?